src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -325,6 +349,14 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
 
                     // filter out all props that cannot be edited or are link props
                     this.properties = onto.getPropertyDefinitionsByType(ResourcePropertyDefinition).filter(prop => prop.isEditable && !prop.isLinkProperty);
+
+                    // notifies the user that the select resource has got 0 property yet defined.
```

Comment:
notifies the user that the selected resource does not have any properties defined yet.

Reply:
fixed in 3717fc5